index.php

```diff
@@ -9,22 +9,51 @@
 <body>
 
   <h1>History</h1>
-
-  <h2>Previous periods:</h2>
+  <h2>Assume your cycle length is <span style = 'color: red;'> 30 days </span></h2>
 
       <?php
-      #good to make this a seperate function later. thoughts?
-      $dates = dbQuery(' 
-          SELECT startDate, endDate 
-          FROM periods
-        ')->fetchAll();
-
-      foreach ($dates as $row) {
-        echo "Started on ".convertDateToMDY($row['startDate']).". Ended on ".convertDateToMDY($row['endDate'])."<br><br>";
+
+    ##################################################
+    //           PLAN FOR PERIOD DATA:
+
+    ## 1: fetch all period dates from SQL -> returns dates in form of array 
+    ## 2: convert dates to DateTime objects 
+    ## 3: find difference in two dates: today and last period start date 
+
+    ##################################################
+
+    
+    #returns query of dates as array of strings
+    $periodDates = fetchAllPeriodDates();
+    
+    #display all period dates
+    foreach ($periodDates as $row) {
+        $periodStart = turnStringToDTO($row['startDate']);
+        $periodEnd = turnStringToDTO($row['endDate']);
+        echo "Started on ".convertDateTimeToMDYFormat($periodStart).". 
+        Ended on ".convertDateTimeToMDYFormat($periodEnd)."<br><br>";
       }
-      ?>
 
-  <h2>Your next period is coming in X days. </h2>
+    #end() array function
+    $lastPeriod = end($periodDates);
```

Comment:
I would rename the function **_end_** to something more descriptive like **_GetLastPeriodDate_** or something